src/halo/conf/cluster.yaml.sample

```diff
@@ -0,0 +1,64 @@
+cortx:
```

Comment:
Remove cortx and replace with 'root' or 'config' or something generic. Code also will look for either cortx or xxx.

Reply:
okay